dev/io.openliberty.microprofile.reactive.messaging.internal_fat/fat/src/io/openliberty/microprofile/reactive/messaging/fat/kafka/metrics/MetricsTest.java

```diff
@@ -48,6 +49,7 @@
  *
  */
 @RunWith(FATRunner.class)
+@AllowedFFDC("javax.management.InstanceNotFoundException")
 public class MetricsTest extends FATServletClient {
```

Comment:
Given this is the runs on RM 30 onwards, this will throw jakarta.* exceptions, not javax, so this wouldn't stop a failure in the case of the issue

Reply:
Good spot, will update now